src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -558,7 +589,7 @@ protected virtual MvcForm CreateForm()
             return new HtmlString(elements);
         }
 
-        protected virtual string GenerateDisplayName([NotNull] ModelMetadata metadata, string htmlFieldName)
+        protected virtual string GenerateDisplayName([NotNull] ModelMetadata metadata, string expression)
```

Comment:
it _is_ an expression (comments are about use of `ModelMetadata`) and callers all call it either `expression` or `expressionText` (where the method already has an `expression` parameter).  so I just changed this to be consistent w/ callers.